gcore/src/prog.rs

```diff
@@ -71,6 +81,87 @@ mod sys {
 ///     let submitted_code: CodeHash =
 ///         hex_literal::hex!("abf3746e72a6e8740bd9e12b879fbdd59e052cb390f116454e9116c22021ae4a")
 ///             .into();
+///     let new_program_id = prog::create_program(submitted_code, &get().to_le_bytes(), b"", 0);
```

Comment:
I don't really like idea to copy-paste code of some contract. I prefer to insert just usage of the method with snips around. Such huge docs seems unreadable for me.

Reply:
I'm okay with both, since if users have `rls` in their editor, they can see the full doc of both gasful and gasless functions